api/system-status/read

```diff
@@ -73,10 +111,10 @@ function readRelease()
 }
 
 function readKernel()
-    {
```

Comment:
Please remove this whitespace-only change!

Reply:
Removed.